README.md

```diff
@@ -56,6 +56,10 @@ $ meteor add universe:i18n
 
 ### Typescript
 
+This package is integrated with [zodern:types](https://github.com/zodern/meteor-types) package. If you use `zodern:types` package in your project, it will automatically download the current type definitions.
```

Comment:
```suggestion
This package is integrated with [`zodern:types`](https://github.com/zodern/meteor-types) package. If you use `zodern:types` package in your project, it will automatically download the current type definitions.
```